.settings/org.eclipse.jdt.apt.core.prefs

```diff
@@ -1,4 +1,4 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.apt.aptEnabled=true
-org.eclipse.jdt.apt.genSrcDir=target/generated-sources/annotations
-org.eclipse.jdt.apt.genTestSrcDir=target/generated-test-sources/test-annotations
+org.eclipse.jdt.apt.genSrcDir=target\\generated-sources\\annotations
```

Comment:
It looks like this line and the next were Windows-ized, with / path separators replaced by \\ characters.  Could you change this back, please?

Reply:
I'm already on it. got commited by mistake.